pkg/generate/config/config.go

```diff
@@ -39,6 +39,12 @@ type Config struct {
 	// SetManyOutput function fails with NotFound error.
 	// Default is "return nil, ackerr.NotFound"
 	SetManyOutputNotFoundErrReturn string `json:"set_many_output_notfound_err_return,omitempty"`
+	// ModelName lets you specify the path used to identify the AWS service API
```

Comment:
nit: apply your comment for `ServiceModelName` here-- I thought it was very clear!

`// ModelName lets you specify the exact string used to identify the AWS service API...`